Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Art/GetByStreetCodeId/GetArtByStreetcodeIdTest.cs

```diff
@@ -0,0 +1,163 @@
+﻿using AutoMapper;
+using FluentResults;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.Media.Images;
+using Streetcode.BLL.MediatR.Media.Art.GetByStreetcodeId;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System.Linq.Expressions;
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatR.Media.Art.GetByStreetCodeId
+{
+    public class GetImageByStreetcodeIdTest
+    {
+        private Mock<IRepositoryWrapper> _mockRepo;
+        private Mock<IMapper> _mockMapper;
+
+        public GetImageByStreetcodeIdTest()
+        {
+            _mockRepo = new Mock<IRepositoryWrapper>();
+            _mockMapper = new Mock<IMapper>();
+        }
+
+        [Theory]
+        [InlineData(1)]
+
+        public async Task GetArtByStreetCodeId_ReturnsSuccessfullyArtById(int streetcodeId)
+        {
+            var allArtsList = new List<DAL.Entities.Media.Images.Art>()

```

Comment:
Is this class name a Method name? In that case you should place here only tests for that class, If not It should be the name of class you are writing tests
Please check and fix the name conventions that you follow in your tests